console-subscriber/src/init.rs

```diff
@@ -0,0 +1,104 @@
+use crate::TasksLayer;
+use std::thread;
+use tokio::runtime;
+use tracing_subscriber::{fmt, layer::Layered, prelude::*, EnvFilter, Registry};
+
+type ConsoleSubscriberLayer = Layered<TasksLayer, Layered<EnvFilter, Registry>>;
```

Comment:
nit (and this isn't _that important_, since this isn't part of the public API), i would just call this 
```suggestion
type ConsoleSubscriber = Layered<TasksLayer, Layered<EnvFilter, Registry>>;
```
this type _isn't_ actually a `Layer`, since it includes a subscriber --- it doesn't implement the `Layer` trait, it implements the `Subscriber` trait.